sydent/replication/peer.py

```diff
@@ -24,7 +23,7 @@
 import json
 
 import twisted.internet.reactor
-import twisted.internet.defer
```

Comment:
`twisted.internet.defer` appears in a number of places, so I am not sure this is a safe change.

Reply:
Was due to it being in there twice.